<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineManagementWidget</class>
 <widget class="QWidget" name="EngineManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Default location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_defaultLocationEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_browseDefaultLocationBtn">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Configured chess engines:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="m_searchEngineEdit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="m_enginesList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_addBtn">
       <property name="toolTip">
        <string>Add a new engine</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../res/icons/icons.qrc">
         <normaloff>:/icons/toolbutton/add_16x16.png</normaloff>:/icons/toolbutton/add_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove selected engine(s)</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../res/icons/icons.qrc">
         <normaloff>:/icons/toolbutton/remove_16x16.png</normaloff>:/icons/toolbutton/remove_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_configureBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Configure selected engine(s)</string>
       </property>
       <property name="text">
        <string>Configure</string>
       </property>
       <property name="icon">
        <iconset resource="../res/icons/icons.qrc">
         <normaloff>:/icons/toolbutton/configure_16x16.png</normaloff>:/icons/toolbutton/configure_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_cloneBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Clone &amp; modify selected engine configuration</string>
       </property>
       <property name="text">
        <string>Clone</string>
       </property>
       <property name="icon">
        <iconset resource="../res/icons/icons.qrc">
         <normaloff>:/icons/toolbutton/clone_16x16.png</normaloff>:/icons/toolbutton/clone_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_engineCountLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
